packages/MSBot/src/msbot-clone-services.ts

```diff
@@ -59,25 +59,25 @@ interface ICloneArgs {
 
 program
     .name('msbot clone services')
-    .option('-n, --name <name>', 'name of new bot')
-    .option('-f, --folder <folder>', 'path to folder containing exported resources')
-    .option('-l, --location <location>', 'location to create the bot service in (westus, ...)')
-    .option('--luisAuthoringKey <luisAuthoringKey>', 'authoring key from the appropriate luisAuthoringRegion for LUIS resources')
+    .option('-n, --name <name>', 'Name of new bot')
+    .option('-f, --folder <folder>', 'Path to folder containing exported resources')
+    .option('-l, --location <location>', 'Location to create the bot service in (westus, ...)')
+    .option('--luisAuthoringKey <luisAuthoringKey>', 'Authoring key from the appropriate luisAuthoringRegion for LUIS resources')
     .option('--luisAuthoringRegion <luisAuthoringRegion>', '(OPTIONAL) [westus|westeurope|australiaeast] authoring region to put LUIS models into (default is based on location)')
-    .option('--luisPublishRegion <luisRegion>', '(OPTIONAL) region to publish LUIS models to (default fallback is based location || luisAuthoringRegion)')
+    .option('--luisPublishRegion <luisRegion>', '(OPTIONAL) Region to publish LUIS models to (default fallback is based location || luisAuthoringRegion)')
     .option('--subscriptionId <subscriptionId>', '(OPTIONAL) Azure subscriptionId to clone bot to, if not passed then current az account will be used')
-    .option('--insightsRegion <insightsRegion>', '(OPTIONAL) region to create appInsights account in (default is based on location)')
+    .option('--insightsRegion <insightsRegion>', '(OPTIONAL) Region to create appInsights account in (default is based on location)')
     .option('--groupName <groupName>', '(OPTIONAL) groupName for cloned bot, if not passed then new bot name will be used for the new group')
```

Comment:
If you are going to capitalize the start of command documentation lines then do it to them all

Reply:
groupName is an arg, which is why I left it as is.

Reply:
what about language below it?